### ISSUE TITLE
SKIP token in EXT-X-KEY URI 

### ISSUE THREAD

**Issue by anish10dec** [kaltura/nginx-secure-token-module#98]
Is there option to not add hmac token in query parameter of EXT-X-KEY URI .
This is creating problem by sending request of AES Key with each and very chunk , even though key rotation is set to 10 min,

Is there directive to not add hmac token with AES Key URL in manifest file

**Comment by erankor** [comment]
No, unfortunately, there isn't, there's only an option to avoid tokenizing the segment URLs
